plugins/GeoIp2/GeoIP2AutoUpdater.php

```diff
@@ -235,7 +240,7 @@ public static function unzipDownloadedFile($path, $dbType, $unlink = false)
                 'loc' => array(),
                 'isp' => array()
             );
-            $customDbNames[$dbType] = array($tempFilename);
+            $customDbNames[$dbType] = $outputPath;
```

Comment:
don't know how this worked before... had to remove the array as in the PHP provider it accesses that key directly.